.github/workflows/pytest.yaml

```diff
@@ -7,29 +7,28 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pipenv flake8 wheel
-      - id: cache-pipenv
+      - id: cache-poetry
         uses: actions/cache@v1
         with:
-          path: ~/.local/share/virtualenvs
-          key: ${{ runner.os }}-pipenv-${{ hashFiles('**/Pipfile.lock') }}
-      - name: Install dependencies
-        if: steps.cache-pipenv.outputs.cache-hit != 'true'
-        run: |
-          pipenv install --deploy --dev
+          path: ~/.local
+          key: ${{ runner.os }}-python-${{ matrix.python-version }}-poetry-${{ hashFiles('**/poetry.lock') }}
       - name: Test with pytest
         run: |
-          pipenv run pytest -svvv
+          poetry install
+          poetry run pytest -svvv
```

Comment:
Using [pytest-cov](https://pytest-cov.readthedocs.io/en/latest/) we can modify this step so we generate a test coverage report as well:

```
pytest --cov-report=term --cov-report=html:htmlcov --cov=example-data
```

Reply:
If we want to use pytest-cov, we have to install the library. :)

Reply:
When I run `poetry run pytest -svvv` locally, I get the following warning. I know linkml/rdflib compatibility has been an ongoing topic, so maybe it doesn't require any corrective action on our part?

> head-and-mouth/test_load.py::test_transform_gdc_data
  /Users/MAM/Library/Caches/pypoetry/virtualenvs/crdch-example-workflows-ma-Vv354-py3.9/lib/python3.9/site-packages/rdflib_jsonld/__init__.py:9: DeprecationWarning: The rdflib-jsonld package has been integrated into rdflib as of rdflib==6.0.0.  Please remove rdflib-jsonld from your project's dependencies.
    warnings.warn(

Reply:
I don't see flake8 in `pyproject.toml`. How is it supposed to get installed?

Reply:
Can pytest-cov be used to test the coverage of the crdch_model code? Because I think that would be super-useful! I'll open an issue to investigate that: https://github.com/cancerDHC/example-data/issues/46

@turbomam Good catch on flake8! I've added it in ecd2bb7.